### PR TITLE
Term entry cybersecurity phishing

### LINKED ISSUE #2009: [Concept Entry] Cybersecurity Phishing
### Reason/inspiration (optional)

We would like a new term entry for [Cybersecurity](https://www.codecademy.com/resources/docs/cybersecurity) about Phishing

The file should be located at `docs/content/cybersecurity/concepts/phishing/phishing.md`

The entry should:

- Describe the concept of phishing. 
- Explain baisc countermeasures to fight phishing. 
- Mention the difference between phishing and [spear phishing](https://www.codecademy.com/resources/docs/cybersecurity/spear-phishing) or whaling.. 
- Link mentions of other cybersecurity concepts to their entries where appropriate.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

-Added a new term for Cybersecurity: Phishing
-Closes #2009 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cybersecurity/concepts/phishing/phishing.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- Damage to personal or business reputation.
+
+## Types of Phishing
+
+Phishing can happen through any communication channel, and usually, the attacker pretends to be a person or organization the user recognizes or trusts. The following table contains the different types of phishing techniques:
+
+| Type of Phishing          | Definition |
+| --------------------------| -----------|
+| Bulk Phishing Emails      | Deceptive messages sent to a large number of recipients, impersonating legitimate organizations or people to obtain sensitive information or infecting devices with malware. This is the most commonly used type of phishing technique. |
+| [Spare Phishing](https://www.codecademy.com/resources/docs/cybersecurity/spear-phishing)             | Personalized, fraudulent emails that trick users into revealing sensitive information or performing malicious actions. |
```
Comment (caupolicandiaz) [inline content/cybersecurity/concepts/phishing/phishing.md:32]:
```suggestion
| [Spear Phishing](https://www.codecademy.com/resources/docs/cybersecurity/spear-phishing)             | Personalized, fraudulent emails that trick users into revealing sensitive information or performing malicious actions. |
```

File: content/cybersecurity/concepts/phishing/phishing.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+| Business Email Compromise | Manipulated or compromised legitimate email accounts within a business or organization that tries to deceive employees or customers into sharing sensitive information or making financial transactions. |
+| SMS Phishing              | Phishing through SMS or phone text messages. |
+| Voice Phishing            | Phishing through phone calls or voice messages. |
+| Social Media Phishing     | Impersonated or compromised accounts of individuals or organizations on social media platforms that try to trick users into revealing personal information or manipulating them into downloading harmful content. |
+| App or In-App Phishing    | Phishing that occurs within a mobile or web application through deceptive fraudulent interfaces. |
+
+## Signs of Phishing
+
+Although phishing emails or messages can be very decisive, certain characteristics of phishing can help identify fraudulent content. These characteristics are:
```
Comment (caupolicandiaz) [inline content/cybersecurity/concepts/phishing/phishing.md:41]: "decisive" feels a little strange here to my ear. Is the intention to convey that the email can seem direct and authoritative? Or that they can take many forms, there's a diversity of campaigns or approaches, however, they often share these signatures/characteristics?

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla Looks good, I just left a couple of comments.

File: content/cybersecurity/concepts/phishing/phishing.md (hunk 2 above)
Comment (cslylla) [reply content/cybersecurity/concepts/phishing/phishing.md]: Maybe "decisive" is not the right word. I meant something like "difficult to recognize" or "can be very convincing". I rephrased the sentence without "decisive".

**Comment by cslylla** [comment]
@caupolicandiaz I made some changes based on your comments 👍🏻

File: content/cybersecurity/concepts/phishing/phishing.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  - 'Computer Science'
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Phishing** is a form of fraud which aims to trick users into sharing sensitive information or personal data or downloading malware. Phishing can happen through many mediums, including emails, text messages, voice messages, phone calls, or social media. Phishing is one of the most common types of [cyber attacks](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack), and it relies on [social engineering](https://www.codecademy.com/resources/docs/cybersecurity/social-engineering) to defraud users.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/phishing/phishing.md:14]:
```suggestion
**Phishing** is a form of fraud which aims to trick users into sharing sensitive information or personal data or downloading [malware](https://www.codecademy.com/resources/docs/cybersecurity/malware). Phishing can happen through many mediums, including emails, text messages, voice messages, phone calls, or social media. Phishing is one of the most common types of [cyber attacks](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack), and it relies on [social engineering](https://www.codecademy.com/resources/docs/cybersecurity/social-engineering) to defraud users.
```

File: content/cybersecurity/concepts/phishing/phishing.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+- Awareness and continued education on phishing techniques,
+- Staying alert and being cautious before taking actions (e.g. clicking links or providing information),
+- Regularly installing software updates,
+- Frequently changing passwords.
+
+The following tools can support users in preventing phishing:
+
+- Spam filters,
+- Antivirus and anti-malware software,
+- Multi-factor authentication,
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/phishing/phishing.md:67]:
```suggestion
- [Multi-factor authentication](https://www.codecademy.com/resources/docs/cybersecurity/multi-factor-authentication),
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cslylla I did a second review and just saw a couple more opportunities for cross-linking.

**Comment by cslylla** [comment]
@SSwiniarski Thank you for adding the links! I accepted the changes.

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good to merge.

**Comment by SSwiniarski** [comment]
@cslylla Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/cybersecurity/phishing

image

**Pull request merged**